tmt/steps/provision/__init__.py

```diff
@@ -1918,45 +2045,147 @@ def go(self, force: bool = False) -> None:
 
         # Provision guests
         self._guests = []
-        save = True
-        self.is_multihost = sum(isinstance(phase, ProvisionPlugin) for phase in self.phases()) > 1
-        try:
-            for phase in self.phases(classes=(Action, ProvisionPlugin)):
-                try:
-                    if isinstance(phase, Action):
-                        phase.go()
 
-                    elif isinstance(phase, ProvisionPlugin):
-                        phase.go()
+        def _run_provision_phases(
```

Comment:
Please, add a docstring, the return value is not very obvious.

Reply:
Added in db0510dc